VideoAPI/Video.API/Controllers/ConferenceController.cs

```diff
@@ -121,7 +121,7 @@ public async Task<IActionResult> BookNewConferenceAsync(BookNewConferenceRequest
 
             _logger.LogInformation("Created conference {ResponseId} for hearing {HearingRefId}", response.Id, request.HearingRefId);
 
-            return CreatedAtAction(nameof(GetConferenceDetailsByIdAsync), new {conferenceId = response.Id}, response);
+            return CreatedAtAction("GetConferenceDetailsById", new {conferenceId = response.Id}, response);
```

Comment:
is this correct?

Reply:
Yup, this was throwing a non-compiler error warning me this would fail in runtime. "Async" is ignored and since Async is in the name of the method "nameOf" would return a string that CreatedAtAction does not like .

Before:

![image](https://user-images.githubusercontent.com/41630528/104212840-8e254200-542d-11eb-91a9-1047c93a1571.png)


after

![image](https://user-images.githubusercontent.com/41630528/104212862-954c5000-542d-11eb-81d7-3ad147a4f0fb.png)